src/PrivateKey.cpp

```
@@ -120,6 +84,8 @@ PrivateKey::PrivateKey(const Data& data, TWCurve curve) {
     }
     bytes = data;
     _curve = curve;
+    auto* privkey = Rust::tw_private_key_create_with_data(data.data(), data.size(), static_cast<uint32_t>(curve));
+    _impl = privkey ? Rust::wrapTWPrivateKey(privkey) : nullptr;
```

Not fixed yet. Let's throw an exception if `privkey` is null